ddp-workspace/projects/ddp-singular/src/app/components/pages/survey/survey.component.ts

```diff
@@ -57,8 +57,6 @@ export class SurveyComponent implements OnInit, OnDestroy {
       }
       this.workflowBuilder.getCommand(activityResponse).execute();
     });
-
-      this.scrollToTop();
```

Comment:
Removed as not needed. 
see https://github.com/broadinstitute/ddp-angular/pull/1628/files#r937944178